terra/context_processors.py

```diff
@@ -0,0 +1,22 @@
+from datetime import date
+from django.conf import settings
+
+import fiscalyear as FY
+import locale
+
+
+FY.START_MONTH = 7
+
+
+def current_fiscal_year(today=None):
```

Comment:
@aprigge Most/all of the code here duplicates code which is already in `utils.py`.  Is that necessary, or can that code be reused?

Reply:
@akohler, Yes, changed this to import from utils instead of repeating the code.